target/linux/ramips/dts/mt7621_mofinetwork_mofi5500-5gxelte.dts

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "mofinetwork,mofi5500-5gxelte", "mediatek,mt7621-soc";
+	model = "MoFi Network MOFI5500-5GXeLTE";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "green:power";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		internet {
+			label = "green:internet";
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+
+		module_1 {
+			label = "blue:module_1";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		module_2 {
+			label = "blue:module_2";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys-polled";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&switch0 {
+	ports {
+		port@0 {
+			status = "okay";
+			reg = <0>;
+			label = "lan1";
+		};
+
+		port@1 {
+			status = "okay";
+			reg = <1>;
+			label = "lan2";
+		};
+
+		port@2 {
+			status = "okay";
+			reg = <2>;
+			label = "lan3";
+		};
+
+		port@3 {
+			status = "okay";
+			reg = <3>;
+			label = "lan4";
+		};
+
+		port@4 {
+			status = "okay";
+			reg = <4>;
+			label = "wan";
+		};
+	};
+};
+
+&sdhci {
+	status = "okay";
+};
+
+&i2c {
+	status = "okay";
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie0 {
+	wifi0: wifi@0,0 {
```

Comment:
Are the labels `wifi0` and `wifi1` used anywhere?

Reply:
Not sure. I copied it from a different DTS. Maybe OpenWRT uses it for something?

Reply:
If they're not referenced in this DTS, most likely no. Such labels are more common in `dtsi` files, included by other files, or in cases you reference them yourself (c.f., `led_power`).